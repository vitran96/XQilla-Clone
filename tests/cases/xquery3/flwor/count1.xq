for $a in (1 to 100)
order by $a descending
where $a mod 7 = 0
let $b := $a * $a
count $p
where $p mod 2 = 0
return <result p="{$p}" a="{$a}" b="{$b}"/>