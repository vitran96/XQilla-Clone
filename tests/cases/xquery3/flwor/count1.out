<result p="2" a="91" b="8281"/><result p="4" a="77" b="5929"/><result p="6" a="63" b="3969"/><result p="8" a="49" b="2401"/><result p="10" a="35" b="1225"/><result p="12" a="21" b="441"/><result p="14" a="7" b="49"/>